Himanshu-harry6003 modified footer

harry6003 opened the pull request:
Modified Footer With amazing CSS.

When you hover it, it glows like the color of the platform.

![image](https://user-images.githubusercontent.com/72691698/126470651-b5a3b3e8-104f-4a8d-b914-3ccc1dd0896a.png)

tiluckdave commented:
Hey seems good.
will you please mention how many and which issues you have solved?
as this seems more than the footer.

harry6003 commented:
I had just changed the footer, It is looking more because of (prettier) it is vs code extension which makes indentation proper in vs code, I just removed the previous footer and modified only that noting else.

harry6003 commented:
see in this, the content is the same, it just changes the formatting of the code.
![image](https://user-images.githubusercontent.com/72691698/126474643-2254d243-81fd-40fe-a2a6-3fe1c9efc39c.png)

Divyansh1011 commented:
@harry6003 specify the issue no

Divyansh1011 commented:
it will link the issues then

Divyansh1011 commented:
with an hashtag

tiluckdave commented:
OK Seems good.
Well done. we will review your issue soon.

harry6003 commented:
#5 (Make a good footer )

harry6003 commented:
Thanks @tiluckdave & @Divyansh1011

tiluckdave commented:
Hey @harry6003 
Please leave your discord ID so we can contact you later.

harry6003 commented:
Himanshu#2140 (Himanshu Agrawal )
